### ISSUE TITLE
[Feature request] change administration page from port 80 to another port

### ISSUE THREAD

**Issue by rapdodge** [AdguardTeam/AdGuardHome#629]
This feature allows administrators to change port 80 (used by administration pages) to other ports (like 8080, etc)

**Comment by ameshkov** [comment]
You can change it in `AdGuardHome.yaml` already.

So do you mean that you'd like us to let you change these settings in the UI?

**Comment by rapdodge** [comment]
> 
> 
> You can change it in `AdGuardHome.yaml` already.
> 
> So do you mean that you'd like us to let you change these settings in the UI?

Yes, a GUI version like we add filter, disable filter, and turn on the filter, for easy way.

**Comment by Sil3ntVip3r** [comment]
Current `master` already supports selecting a non-default administration port
during initial setup, so I believe this request is implemented and can be
closed.

Evidence at `5f6cb57df9d54d8a89afc9200d813411f9ce4f7a`:

- The installer exposes an editable numeric `install_web_port` field and
  validates the selected port.
- The submitted value is persisted as
  `config.HTTPConfig.Address = netip.AddrPortFrom(req.Web.IP, req.Web.Port)`.
- The installer E2E setup explicitly fills `install_web_port` with its test
  server's non-default `PORT`, exercising the intended workflow.

I also ran the current backend package with race detection:

```text
go test -race -count=1 ./internal/home
ok github.com/AdguardTeam/AdGuardHome/internal/home 11.832s
```

For an existing installation, the same bind address remains configurable
through `http.address` in `AdGuardHome.yaml` while the service is stopped.

**Comment by Sil3ntVip3r** [comment]
Correction to my previous comment: I interpreted the request too broadly.
The installer does allow choosing the initial administration port, but the
reporter explicitly confirmed that this issue asks for a **post-install GUI
setting**.  I rechecked current `master` (`369d4761`) and found no dashboard
control or control API for changing `http.address`; an existing installation
still requires editing `AdGuardHome.yaml` (or using the startup option) and
restarting the service.

So the original UI feature request is not fully implemented and should remain
open unless the maintainers decide that installer/config-file support is the
intended scope.  Sorry for the incorrect closure recommendation.
